bsp/boards/iot-lab_A8-M3/board.c

```diff
@@ -98,12 +101,40 @@ void board_init(void){
     debugpins_init();
     //enable nvic for the radio
     NVIC_radio();
-
+    board_init_slot_vars();
 #if defined(BOARD_CRYPTOENGINE_ENABLED)
     cryptoengine_init();
 #endif
 }
 
+
+//==== bootstrapping slot info lookup table
+void board_init_slot_vars(void){
+    

```

Comment:
Initialize the slot_board_vars variable here, i.e. using the memset(&slot_board_vars[0], 0, sizeof(slot_board_vars));

Need to check the value of sizeof(slot_board_vars).